client/src/components/UploadFileButton/UploadFileButton.tsx

```diff
@@ -1,12 +1,28 @@
 import { Box, Button } from '@material-ui/core';
 import { useStyles } from './useStyles';
 import { IUploadFile } from '../../interface/UploadFileButton';
+import { ChangeEvent, useState } from 'react';
+import { uploadImage } from '../../helpers/APICalls/uploadImg';
+import { useSnackBar } from '../../context/useSnackbarContext';
 
 export default function UploadFileButton({ ...inputProps }: IUploadFile): JSX.Element {
   const classes = useStyles();
+  const URL = 'https://api.cloudinary.com/v1_1/dog-sitter/image/upload';
+  const [image, setImage] = useState<File | undefined>(undefined);
+  const { updateSnackBarImage } = useSnackBar();
+
+  const onChange = (event: ChangeEvent) => {
+    const target = event.target as HTMLInputElement;
+    const file: File = (target.files as FileList)[0];
+    if (file) {
+      setImage(file);
+      uploadImage(URL, image).then((data: any) => updateSnackBarImage(data.secure_url));
```

Comment:
```suggestion
      uploadImage(URL, image)
      .then((data: ImageResponseApiData) => updateSnackBarMessage("Image has been successfully uploaded"))
      .catch((err) => updateSnackBarMessage(err.message))
```

Maybe consider using an interface for response data type rather than using 'any' 😄